ntc_templates/templates/cisco_ios_show_power_inline.textfsm

```diff
@@ -0,0 +1,24 @@
+Value Interface (\S+)
+Value Admin (\S+)
+Value Oper (on|off)
+Value Power (\d+\.\d+)
+Value Device (\S+(\s\S+)*)
+Value Class (\d|n/a)
+Value Max (\d+\.\d+)

```

Comment:
```suggestion
Value INTERFACE (\S+)
Value ADMIN_STATUS (\S+)
Value OPERATIONAL_STATUS (on|off)
Value POWER (\d+\.\d+)
Value DEVICE (\S+(\s\S+)*)
Value CLASS (\d|n/a)
Value MAX (\d+\.\d+)

```
We standardize on using all caps for capture group names for ease of identifying, and started standardizing some of the names across all templates